redux/store.ts

```diff
@@ -28,4 +28,4 @@ const makeStore: MakeStore<State> = (context: Context) => {
   );
 };
 
-export const wrapper = createWrapper<State>(makeStore, { debug: true });
+export const wrapper = createWrapper<State>(makeStore, { debug: false });
```

Comment:
I changed this to prevent filling the console with debugging logs. If you need to debug the state, just turn it back on 🙂